cpp/include/tensorrt_llm/batch_manager/cacheTransceiver.h

```diff
@@ -202,7 +202,8 @@ class BaseCacheTransceiver
     virtual void requestAndReceiveSync(LlmRequest* llmRequest) = 0;
     virtual void requestAndReceiveAsync(LlmRequest* llmRequest) = 0;
 
-    virtual void checkContextTransferStatus(std::optional<int> const& atLeastRequestNum = std::nullopt) = 0;
+    virtual std::tuple<std::vector<LlmRequest::RequestIdType>, std::vector<LlmRequest::RequestIdType>>
```

Comment:
could we add some docs about what the return type is supposed to represent? thanks